api/.env.dev

```diff
@@ -21,6 +21,8 @@ JWT_SIGNING_KEY="TektonHub"
 ACCESS_JWT_EXPIRES_IN="1d"
 REFRESH_JWT_EXPIRES_IN="1d"
 
+CATALOG_REFRESH_INTERVAL="30m"
```

Comment:
this will be required in api configmap and deployment right?

Reply:
yes